MediaBrowser.Api/Images/ImageService.cs

```diff
@@ -606,6 +607,12 @@ public Task<object> GetImage(ImageRequest request, Guid itemId, BaseItem item, b
             IDictionary<string, string> headers,
             bool isHeadRequest)
         {
+            if (!image.IsLocalFile)
+            {
+                item ??= _libraryManager.GetItemById(itemId);
+                image = await _libraryManager.ConvertImageToLocal(item, image, request.Index ?? 0).ConfigureAwait(false);
```

Comment:
why? especially given that this PR is aimed at rectifying DI

Reply:
I was eliminating an unnecessary circular dependency in the DI framework by eliminating `_libraryManager` from `ImageProcessor`. See my comment [here](https://github.com/jellyfin/jellyfin/pull/2765/files#r403638431).

This logic was moved from the start of `ImageProcessor.ProcessImage` to here. You can see  that `ImageProcessor.ProcessImage` is called [a few lines below this](https://github.com/jellyfin/jellyfin/pull/2765/files#diff-94d9c9c6f3144cc37f203fc619673b63R637) and this is the only place that method is called so it's safe to pull that logic up on level into this method.

Reply:
But I don't see that call removed anywhere. So, taking what you said, we now have this code at two places?

Reply:
Sorry, I also refactored it when I moved it. It's removed from [here](https://github.com/jellyfin/jellyfin/pull/2765/files#r403638431) like I said, but it looks a bit different